filebeat/module/elasticsearch/_meta/kibana/default/dashboard/elasticsearch-logs.json

```diff
@@ -0,0 +1,19 @@
+[
```

Comment:
@joshuar These dashboard files don't seem to have been exported correctly. How did you do it? Best would be to do it like this: https://www.elastic.co/guide/en/beats/devguide/current/export-dashboards.html#_export_all_module_beat_dashboards

Or were they created on a 5.x instance?

Reply:
I think I can recover them from these files if needed, but if you can export them by following that guide it's going to be easier.

Reply:
@tsg ah yep, lemme export them the proper way.  I'll update the PR next week.